elastic/apm-server
# packaging: fix /licenses in Docker images

## Motivation/summary

Fix COPY to copy licenses to a /licenses/ directory, rather than creating a file called /licenses.

## Checklist

~- [ ] Update [CHANGELOG.asciidoc](https://github.com/elastic/apm-server/blob/main/CHANGELOG.asciidoc)~
~- [ ] Update [package changelog.yml](https://github.com/elastic/apm-server/blob/main/apmpackage/apm/changelog.yml) (only if changes to `apmpackage` have been made)~
~- [ ] Documentation has been updated~

## How to test these changes

1. `make package-docker`
2. `docker run --rm <image> ls /licenses`

There should be license and notice files there.

## Related issues

None
